components/AddFundsSourcePicker.js

```diff
@@ -56,28 +56,23 @@ class AddFundsSourcePickerForUser extends React.Component {
     this.state = { loading: true };
   }
 
-  onChange = async e => {
-    this.props.onChange(e.target.value);
-  };
-
   render() {
     const hosts = this.props.LoggedInUser.hostsUserIsAdminOf();
+    const hostOptions = [
+      ...hosts.map(h => {
+        const value = h.legacyId || h.id;
+        return { key: `addfsph-${h.id}`, value: value, label: h.name };
+      }),
+    ];
     return (
       <div>
-        <FormControl
+        <StyledSelect
           id="sourcePicker"
-          name="template"
-          componentClass="select"
-          placeholder="select"
-          onChange={this.onChange}
-        >
-          <option value="" key="addfsph-00" />
-          {hosts.map(h => (
-            <option value={h.legacyId || h.id} key={`addfsph-${h.id}`}>
-              {h.name}
-            </option>
-          ))}
-        </FormControl>
+          placeholder="Select"
+          options={hostOptions}
+          isSearchable={false}
+          onChange={this.props.onChange}
```

Comment:
This will pass the full option (`{ label, value }`) to `onChange`. As a result, I'm getting this error when I try to submit:
![image](https://user-images.githubusercontent.com/1556356/104707466-b8704b80-571c-11eb-936e-418a39b6b10d.png)

Reply:
@Betree : Thanks for pointing it out. I think I've made a mistake here. I've corrected it; please take a look. 😄